tests/autoskip/imported/Earthfile

```diff
@@ -0,0 +1,5 @@
+VERSION 0.7
+
+test:
+    FROM alpine
+    RUN echo "hi"
```

Comment:
I think it would be a good idea to extend this to perform a `COPY some-file .`

then verify that editing this file will bust the cache.

Reply:
That should be covered elsewhere, but I suppose it doesn't hurt to do something similar here.

Reply:
yeah my thinking is to be absolutely sure the imported referencing is making its way into the imported Earthfile.

Reply:
Updated.